app/controllers/concerns/cart.rb

```diff
@@ -1,5 +1,5 @@
 module Cart
   def current_order
-    @order = current_user.orders.in_progress.last || Order.create(user_id: current_user.id).in_progress!
+    @order = current_user.orders.in_progress.last || Order.in_progress.create(user_id: current_user.id)
```

Comment:
What if `create` fails here. current_order will return an order that does not exist in database